include/libsemigroups/presentation.hpp

```diff
@@ -1536,8 +1581,8 @@ namespace libsemigroups {
     //! \tparam Word the type of the words in the presentation.
     //! \param p the presentation.
     //!
-    //! \throws LibsemigroupsException if \ref validate throws on the initial
-    //! presentation.
+    //! \throws LibsemigroupsException if \ref throw_if_bad_alphabet_or_rules
```

Comment:
This gives:
```
/home/joseph/Dev/Comp_Maths/libsemigroups/include/libsemigroups/presentation.hpp:1583: warning: unable to resolve reference to 'throw_if_bad_alphabet_or_rules' for \ref command
```

```suggestion
    //! \throws LibsemigroupsException if \ref Presentation::throw_if_bad_alphabet_or_rules
```
might fix this, but will need to be reflowed.